src/main/java/com/google/gcp_variant_transforms/library/VariantToBqUtilsImpl.java

```diff
@@ -190,36 +194,40 @@ public void addFormatAndPhaseSet(TableRow row, Genotype genotype, VCFHeader vcfH
     String phaseSet = "";
     // Iterate all format fields in VCFHeader.
     for (VCFFormatHeaderLine formatHeaderLine : vcfHeader.getFormatHeaderLines()) {
-      String fieldName = formatHeaderLine.getID();
-      if (fieldName.equals(VCFConstants.GENOTYPE_KEY)) {
+      String attrName = formatHeaderLine.getID();
```

Comment:
Please confirm: you replaces `fieldName` with `attrName` to make this function more consistent with `addInfo()`?

If that's the case, great idea. Thanks!

Reply:
Yes.